### PR TITLE
replace custom logic in loadVersions with node-version-data package

### PR DESCRIPTION
**Pull request by rvagg**
I'm encouraging the npm team to use this same package to fetch the data they need for mapping node to npm versions

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
     "metalsmith-serve": "0.0.3",
     "metalsmith-stylus": "1.0.0",
     "ncp": "2.0.0",
+    "node-version-data": "~1.0.0",
```
Comment (evanlucas) [inline package.json]: do we need to pin to an exact version here? It seems like that is the current way of doing it.

**Comment by evanlucas** [comment]
LGTM

File: package.json (hunk 1 above)
Comment (stevemao) [inline package.json]: Yes, we are using greenkeeperio here. example: https://github.com/nodejs/new.nodejs.org/pull/279

Comment (rvagg) [inline package.json]: whoops, didn't realise! fixed

**Pull request merged**